src/collection.js

```diff
@@ -149,18 +149,18 @@ function importChange(transaction, remote) {
     }
     return {
       type: "conflicts",
-      data: {type: "incoming", local: local, remote: remote}
+      data: {type: "incoming", local: local, remote: remote, previous: local}
     };
   }
   if (remote.deleted) {
     transaction.delete(remote.id);
-    return {type: "deleted", data: {id: local.id}};
+    return {type: "deleted", data: {id: local.id}, previous: local};
```

Comment:
For `deleted` we may just want to provide `local` in the `data` attribute instead of just the `id` in data and the full record in `previous` ?